scripts/split-paired-reads.py

```diff
@@ -92,11 +92,13 @@ def main():
 
     infile = args.infile
 
-    check_input_files(infile, args.force)
     filenames = [infile]
     check_space(filenames, args.force)
-
     # decide where to put output files - specific directory? or just default?
+    if os.path.basename(infile) == 'stdin':
+        if not (args.output_first and args.output_second):
+            print >>sys.stderr, 'Output files are missing'
```

Comment:
error message should say Accepting input from stdin; output files must be provided.